src/components/navigation/Sidebar.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import logo from '../../assets/images/logo.png';
+import twitter from '../../assets/images/twitter-icon.png';
+import facebook from '../../assets/images/facebook-icon.png';
+import linkedin from '../../assets/images/linkedin-icon.png';
+import github from '../../assets/images/github-icon.png';
+
+const navigation = [
+  { name: 'Cars', href: '#', current: true },
+  { name: 'Reserve', href: '#', current: false },
+  { name: 'My reservations', href: '#', current: false },
```

Comment:
Hi @GErardus , 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif)

## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._